.github/workflows/code-quality.yml

```
@@ -12,19 +12,49 @@ jobs:
+        run: docker run --rm -i hadolint/hadolint < Dockerfile.n8n
+      - name: Lint Dockerfile.temporal
+        run: docker run --rm -i hadolint/hadolint < Dockerfile.temporal
+
+  service-check:
+    name: Service Availability Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set up Docker Compose
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)